src/main/java/org/jenkinsci/plugins/codesonar/CodeSonarAlertLevels.java

```diff
@@ -0,0 +1,10 @@
+package org.jenkinsci.plugins.codesonar;
+
+import com.google.gson.annotations.SerializedName;
+
+public enum CodeSonarAlertLevels {
```

Comment:
This `CodeSonarAlertLevels` enum should probably live in the `models` subpackage.  As far as I can tell "conditions" depends on "api" and "api" depends on "models".  I think `CodeSonarAlertLevels` is used by "models", so I think it should live there.  Plus, it looks like this class implicitly uses GSON; another reason why "models" is probably more appropriate.